viewer/multies.js

```diff
@@ -115,15 +118,59 @@ function node2Prefix (node) {
   return '';
 }
 
-var activeESNodes = [];
+function removeClusterFromQueryUrl (url) {
```

Comment:
I think this whole function can be replaced with
url = url.replace(/_cluster=[^&]*(&|$)/g, '');